test/pg-db/market_repository_impl_test.go

```diff
@@ -177,3 +177,48 @@ func (s *PgDbTestSuite) TestGetMarketsForFilterWithPagination() {
 
 	s.Equal(1, len(markets))
 }
+
+func (s *PgDbTestSuite) TestActivateInactivateMarket() {
```

Comment:
Can we test here also `GetMarketForActiveIndicator`? If I'm not wrong, we didn't test it at all while we do use it in app layer.

Reply:
Done